src/specs/api.yaml

```diff
@@ -58,53 +58,10 @@ paths:
                   - env
                   - clusterEnv
       description: Returns a status on the health of the API.
-  /workResults:
```

Comment:
Good point! Don't we need to remove some SNS subscriptions for this too? Also remove the SNS subscription for staged environments? How is that being handled?

Reply:
I would do those the same way we deal with other IAC stuff. Simply ignore it until it's merged into production, then remove it. Better for backwards compatibility.

Reply:
Ok, so we have to make a ticket so that we don't forget about it. I will make one

Reply:
Actually, if we fist merge the API code for this without modifying the SNS cloudformation files, wouldn't this cause subscription errors? Wouldn't sns complain that it can't find that endpoint? What happened in staging?

Reply:
Yes, SNS would complain that it can't subscribe, but that does not cause any errors as such. That subscription will just show up as "unsubscribed" in the AWS console. Not sure if that is a big deal.